cluster/v1alpha1/types.go

```diff
@@ -267,6 +271,52 @@ type ClusterClaimSelector struct {
 	MatchExpressions []metav1.LabelSelectorRequirement `json:"matchExpressions,omitempty"`
 }
 
+// PrioritizerConfig represents the configuration of prioritizer
+type PrioritizerConfig struct {
+	// Name is the name of a prioritizer. Below are the valid names:
+	// 1) Balance
```

Comment:
maybe we can do like

Name is the name of a prioritizer. Below are the valid names:
1) Balance: balance the decisions among the cluster.
2) Steady: ensure the existing decision is stabilized.
3) ResourceRatioCPU & ResourceRatioMemory: sort cluster based on the allocatable to capacity ratio.
4) ResourceAllocatableCPU & ResourceAllocatableMemory: sort cluster based on the allocatable.

Reply:
@qiujian16 Thanks. The original idea is, as we expect users to set the weight to some Prioritizer, give more details like score range may help. If it's not necessary, it's good to me to simplify the words.